src/openforms/emails/tests/test_tasks.py

```diff
@@ -399,3 +401,162 @@ def test_no_email_sent_when_brk_congiguration_is_invalid_but_unused(self):
             send_email_digest()
 
         self.assertEqual(0, len(mail.outbox))
+
+    def test_broken_bag_configuration_for_address_fields_is_sent(self):
+        form = FormFactory.create()
+        form_definition = FormDefinitionFactory.create(
+            configuration={
+                "display": "form",
+                "components": [
+                    {
+                        "key": "postCode",
+                        "type": "postcode",
+                        "label": "Postcode",
+                    },
+                    {
+                        "key": "houseNumber",
+                        "type": "textfield",
+                        "label": "house number",
+                    },
+                    {
+                        "key": "streetName",
+                        "type": "textfield",
+                        "label": "Street name",
+                        "deriveCity": False,
+                        "derivePostcode": "postCode",
+                        "deriveStreetName": True,
+                        "deriveHouseNumber": "houseNumber",
+                    },
+                ],
+            }
+        )
+        FormStepFactory.create(form=form, form_definition=form_definition)
+
+        with (
+            freeze_time("2023-01-03T01:00:00+01:00"),
+            patch(
+                "openforms.emails.tasks.GlobalConfiguration.get_solo",
+                return_value=GlobalConfiguration(
+                    recipients_email_digest=["user@example.com"]
+                ),
+            ),
+            patch(
+                "openforms.contrib.kadaster.models.KadasterApiConfig.get_solo",
+                return_value=KadasterApiConfig(bag_service=None),
+            ),
+        ):
+
+            send_email_digest()
+
+        sent_email = mail.outbox[0]
+
+        self.assertEqual(
+            sent_email.subject, "[Open Forms] Daily summary of detected problems"
+        )
+        self.assertEqual(sent_email.recipients(), ["user@example.com"])
+        self.assertIn(
+            _(
+                "The configuration for 'BAG Client' is invalid (bag_service endpoint is not configured)."
+            ),
```

Comment:
This is not a proper translation message - you need to use the same placeholders like in the original template: https://github.com/open-formulieren/open-forms/blob/3f6e71284b92eefe014a82d3f5c00bc3d956d076/src/openforms/emails/templates/emails/admin_digest.html#L72

Something along the lines of:

```suggestion
            _(
                "The configuration for '%(name)s' is invalid (%(message)s)."
            ),
```
but this is flawed anyway, because this exact translation string doesn't exist, as it was created by stripping tags from the HTML. Since you force the language to be english in these tests, skip the marking-translatable here and just do a literal string assertion because you control the language of the message explicitly already:

```suggestion
            "The configuration for 'BAG Client' is invalid (bag_service endpoint is not configured).",
```